sites/all/modules/publizon/includes/publizon.loan.inc

```diff
@@ -111,6 +111,15 @@ function publizon_loan_create($account, $local_id) {
       }
     }
 
+    // The user may belong to an institution that has been removed from the
+    // current institution list (session with old data is still alive).  Don't
+    // allow creating a new loan in this case.
+    if (NULL !== $institution_id
+        && function_exists('ding_unilogin_filter_institutions')
```

Comment:
Will the function not always existe in the codebase. The `module_exists` should be used to check if the module is enabled.

Reply:
Does the function exist if the module is not loaded? I hope not, but I've changed to `module_exists('ding_unilogin')`.